src/AzureStack/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj

```diff
@@ -24,4 +24,5 @@
     <!-- Please do not move/edit code below this line -->
     <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.RP.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
     <!-- Please do not move/edit code above this line -->
+    
```

Comment:
Please undo these changes
in other files too

Reply:
So I should not ever include these?  Some people tell me to add it and others don't.  Please point me towards a csproj that I should match and I will make them all have the same structure.

Reply:
The only thing that changed in this file is adding a newline (not necessary), everything else is okay, all you need to do is revert this change

Reply:
Done, will push when I find out if I should do 0.1.0.0 or 1.0.0.0 in the assembly.